packages/frontend/src/components/common/PivotTable/tableStyles.tsx

```diff
@@ -2,14 +2,26 @@ import { createStyles } from '@mantine/core';
 
 export const useStyles = createStyles((theme) => ({
     table: {
-        '& td, & th': {
+        'td, th': {
             whiteSpace: 'nowrap',
         },
-    },
 
-    header: {
-        fontWeight: 600,
-        backgroundColor: theme.colors.gray[0],
+        td: {
+            transitionProperty: 'color, background-color, border-color',
+            transitionDuration: '100ms',
+            transitionTimingFunction: 'ease-in-out',
+        },
+
+        tbody: {
+            'tr:hover': {
+                td: {
+                    '&:not([data-conditional-formatting="true"]):not([data-expanded="true"])':
```

Comment:
so this is how to do conditional styling! css in the useStyles is the way to go?

Reply:
`useStyles` is one way to stile a component, you can have conditional styles everywhere. best solution is to use style props https://mantine.dev/styles/style-props/ but there are many ways to style a component with mantine

<img width="268" alt="Screenshot 2023-04-04 at 12 28 47" src="https://user-images.githubusercontent.com/962095/229733915-a3da5186-7b00-4455-8587-71fbefb54e68.png">